scss/_alerts.scss

```diff
@@ -24,83 +25,60 @@
     border-color: #d2b2b2;
   }
 
-  .flash-close {
-    float: right;
-    padding: 17px;     // bigger hit area
-    margin-top: -15px; // negative margins to account for bigger padding
-    margin-right: -15px;
-    margin-left: 20px;
-    color: inherit;
-    text-decoration: none;
-    cursor: pointer;
-    opacity: 0.6;
+  // Small flashes
+  &.flash-sm {
+    padding: 8px 16px;
 
-    &:hover {
-      opacity: 1;
+    .btn-sm {
+      margin-top: -2px;
     }
   }
 
-  p:last-child {
-    margin-bottom: 0;
+  // Add an icon
+  > .octicon {
+    float: left;
+    margin-right: 12px;
+    margin-top: 3px;
   }
 
-  // Flashes can have an action button instead of the cross.
-  .flash-action {
+  // Flashes can have an action button
+  .btn {
     float: right;
-    margin-top: -4px;
-    margin-left: 20px;
+    margin-top: -6px;
+    margin-left: 16px;
   }
 
-  // Make links stand out
-  a {
-    font-weight: bold;
+  p:first-child {
+    margin-top: 0;
   }
-}
-
-.flash {
-  padding: 15px;
-  border-radius: 3px;
-
-  + .flash { margin-top: 5px; }
-}
-
-// Add an icon
-.flash-with-icon {
-  padding-left: 40px;
 
-  > .octicon {
-    float: left;
-    margin-top: 3px;
-    margin-left: -25px;
+  p:last-child {
+    margin-bottom: 0;
   }
-}
-
 
-// Global notices
-//
-// Useful for site-wide banners and global notifications.
-
-.flash-global {
-  padding: 10px;
-  margin-top: -1px;
-  border-width: 1px 0;
-
-  h2,
-  p {
-    margin-top: 0;
-    margin-bottom: 0;
-    font-size: 14px;
-    line-height: 1.4;
+  // Make links stand out
+  a {
+    font-weight: bold;
   }
 
-  // Global flashes can have an action buttons too.
-  .flash-action {
-    margin-top: 5px;
+  // More flashes below
+  + .flash {
+    margin-top: 10px;
   }
 }
 
-// Content within flash messages.
-.flash-title {
-  margin-top: 0;
-  margin-bottom: 5px;
+// Close a flash
+.flash-close {
+  color: inherit;
+  cursor: pointer;
+  float: right;
+  margin-top: -16px;
```

Comment:
Properties should be ordered color, cursor, float, margin-right, margin-top, opacity, padding, text-decoration